test_util/integration_test.py

```diff
@@ -21,6 +21,11 @@
 AWS_ACCESS_KEY_ID = os.environ.get('AWS_ACCESS_KEY_ID', '')
 AWS_SECRET_ACCESS_KEY = os.environ.get('AWS_SECRET_ACCESS_KEY', '')
 AWS_REGION = os.environ.get('AWS_REGION', '')
+AUTH_ENABLED = os.environ.get('DCOS_AUTH_ENABLED', 'true') == 'true'
+# If auth is enabled, by default, tests use hard-coded OAuth token
+# Set these to run test against a custom configured user instead
+LOGIN_UNAME = os.environ.get('DCOS_LOGIN_UNAME', '')
```

Comment:
Be better to not specify the second arg here, so we just get the default behavior of None